src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -417,7 +412,7 @@ private BlittableJsonReaderObject WriteNewCountersDocument(DocumentsOperationCon
                     builder.WritePropertyName(DbIds);
 
                     builder.StartWriteArray();
-                    builder.WriteValue(context.Environment.Base64Id);
+                    builder.WriteValue(_localNodeTagAndDbId);
```

Comment:
just db id here. The node tag may change over time, and we don't want to mess things up

Reply:
done